contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1260,19 +1292,21 @@ where:
 <variable_2>:: Specifies the explanation of the first variable.
 ....
 +
-Be sure to introduce the description list with "where:" and start each variable
-description with "Specifies."
+Be sure to introduce the description list with "where:" and start each variable description with "Specifies."
```

Comment:
Should we include an example? 

```
where:

`<project_name>`:: Specifies the name of the project.
`<ip_address>`:: Specifies one or more egress IP addresses for the `egressIPs` array.
```

Should we suggest that the call-out method start with "Specifies" also, as in the example abouve?

Reply:
Sure, that might be helpful. Added one